test/e2e/config/rabbitmq/cluster.yaml

```diff
@@ -22,7 +22,7 @@ spec:
   resources:
     limits:
```

Comment:
Maybe add a comment here that to run local tests with the RabbitMQ Cluster Operator you will need sometimes 800+ of memory (like in Windows WSL for example)